robolectric/src/test/java/org/robolectric/shadows/ShadowAccountManagerTest.java

```diff
@@ -1130,4 +1133,59 @@ T getResult() throws Exception {
       return accountManagerFuture.getResult();
     }
   }
+
+  @Test
+  @Config(minSdk = O)
+  public void
+      accountManager_activityContextEnabled_applicationInstanceIsNotSameAsActivityInstance() {
+    String originalProperty = System.getProperty("robolectric.createActivityContexts", "");
+    System.setProperty("robolectric.createActivityContexts", "true");
+    try {
+      AccountManager applicationAccountManager = appContext.getSystemService(AccountManager.class);
+      Activity activity = Robolectric.setupActivity(Activity.class);
+      AccountManager activityAccountManager = activity.getSystemService(AccountManager.class);
+      assertThat(applicationAccountManager).isNotSameInstanceAs(activityAccountManager);
+    } finally {
+      System.setProperty("robolectric.createActivityContexts", originalProperty);
+    }
+  }
+
+  @Test
+  @Config(minSdk = O)
+  public void accountManager_activityContextEnabled_activityInstanceIsSameAsActivityInstance() {
```

Comment:
I would delete this test, it seems like overtesting.

Reply:
> I would delete this test, it seems like overtesting.

so should I delete both tests?

Reply:
> I would delete this test, it seems like overtesting.

Hi @utzcoz previously, I added the tests, but as suggested by Michael I have removed the tests